moonriver/fast_general_admin/moonriver_pbr_grants_june2024.md

```diff
@@ -0,0 +1,59 @@
+
+# Use of Parachain Bond Reserve to Fund Moonriver Grants (June 2024)
+
+## TL;DR
+In April 2024, a [Forum post](https://forum.moonbeam.network/t/call-for-input-shaping-moonrivers-defi-strategy/1673) solicited community input to shape Moonriver’s DeFi strategy and explore using the parachain bond reserve. Three teams submitted proposals, and after a thorough evaluation, the [Beamswap](https://forum.moonbeam.network/t/beamswap-s-moonriver-ecosystem-grant-draft-proposal/1697) team was selected to receive funding. This proposal requests 96,909 MOVR from the Parachain Bond Reserve to fund:
```

Comment:
Add a link to the comment where the winner was announced: https://forum.moonbeam.network/t/call-for-input-shaping-moonrivers-defi-strategy/1673/15